Library/Formula/cutecom.rb

```diff
@@ -0,0 +1,18 @@
+class Cutecom < Formula
+  homepage "http://cutecom.sourceforge.net"
+  url "http://cutecom.sourceforge.net/cutecom-0.22.0.tar.gz"
+  version "0.22.0"
+  sha1 "a23f9ddf3d1a4467568c872477fc8c6da07c1711"
+
+  depends_on "cmake" => :build
+  depends_on "qt" => ["with-qt3support"]
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "ls", "/Applications/CuteCom.app"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!